src/sentry/features/manager.py

```diff
@@ -31,14 +47,23 @@ def add_handler(self, handler):
         for feature_name in handler.features:
             self._handler_registry[feature_name].append(handler)
 
-    def _get_handler(self, feature, actor):
+    def _get_handler(self, feature: Feature, actor: "User") -> Optional[bool]:
         for handler in self._handler_registry[feature.name]:
             rv = handler(feature, actor)
             if rv is not None:
                 return rv
         return None
 
-    def has_for_batch(self, name, organization, objects, actor=None):
+    def _get_feature_class(self, name: str) -> Type[Feature]:
```

Comment:
Are you sure we need this method in the baseclass? Would make it abstract if so

Reply:
We need this abstract method on this class because the `has_for_batch()` method instantiates a `FeatureCheckBatch` and sets `manager` as `self` as a `RegisteredFeatureManager`.